libnavigation-core/src/main/java/com/mapbox/navigation/core/routerefresh/DirectionsRouteDiffProvider.kt

```diff
@@ -15,16 +16,30 @@ internal class DirectionsRouteDiffProvider {
         val newRouteLegs = newRoute.directionsRoute.legs() ?: return emptyList()
         val routeDiffs = arrayListOf<String>()
         for (legIndex in currentLegIndex until min(oldRouteLegs.size, newRouteLegs.size)) {
-            val oldLegAnnotation = oldRouteLegs[legIndex].annotation()
-            val newLegAnnotation = newRouteLegs[legIndex].annotation()
-            val updatedAnnotations = getUpdatedAnnotations(oldLegAnnotation, newLegAnnotation)
+            val oldLeg = oldRouteLegs[legIndex]
```

Comment:
add please route index, as we'll use the same for alternatives

Reply:
great point. Do you mind if I add it during implementation of route alternatives refresh. I don't exactly know how I will be using routes diff provider.

Reply:
sure, let's then mention it in the original issue